msteams-platform/bots/how-to/add-power-virtual-agents-bot-to-teams.md

```diff
@@ -41,61 +41,55 @@ After publishing your bot, add the Teams channel to make the bot available to Te
 1. **Generate an App ID for your chatbot**  
 After adding the Teams channel to your chatbot, an **App ID** is generated in the dialog box. The App ID is a unique Microsoft generated identifier for your bot. Save the App ID to create an app package for Teams.
 
-## Add your bot to Teams using App Studio
+## Add your bot to Teams using Developer Portal
 
-If [uploading custom apps is enabled](/microsoftteams/admin-settings) in your Teams instance, you can use Teams App Studio to directly upload your chatbot and start using it immediately. To share your chatbot, you can request your admin to make your bot available in the tenant app catalog or you can send your app package to others and ask them to upload it independently.
+If [uploading custom apps is enabled](/microsoftteams/admin-settings) in your Teams instance, you can use Teams Developer Portal to directly upload your chatbot and start using it immediately. To share your chatbot, you can request your admin to make your bot available in the tenant app catalog or you can send your app package to others and ask them to upload it independently.
 
-1. **Install App Studio in Teams**  
-App Studio is a Teams app. Install App Studio from the Teams store that simplifies the process of bot creation and registration in Teams:
+1. **Install Developer Portal in Teams**
 
-   1. Select the app store icon from Teams instance, and search for **App Studio**.
+   Developer Portal is a Teams app. Install Developer Portal from the Teams store that simplifies the process of bot creation and registration in Teams:
```

Comment:
Developer Portal is a Teams app - Aren't we mentioning this above by saying install developer portal? Why do we have this call out again?